java/datastructures/lib/src/main/java/codechallenges/linkedlistinsertion/linkedlistinsertions.java

```diff
@@ -0,0 +1,80 @@
+// append
```

Comment:
Ultimately, these should be part of your LinkedList class, like your original `insert` method.